demos/platforms/stm32f103c8.cpp

```diff
@@ -42,11 +43,19 @@ resource_list initialize_platform()
                                   });
 
   static hal::stm32f1::output_pin led('C', 13);
+  static hal::stm32f1::output_pin sda('B', 7);
```

Comment:
Can you add an issue to remove the "configure" calls here once https://github.com/libhal/libhal-soft/issues/36 is resolved.